pxc/jenkins/pxc56-pipeline.groovy

```diff
@@ -256,7 +256,7 @@ pipeline {
                             sudo git -C sources reset --hard || :
                             sudo git -C sources clean -xdf   || :
 
-                            until aws s3 cp --no-progress s3://pxc-build-cache/${BUILD_TAG}/pxb23.tar.gz ./pxc/sources/pxc/results/pxb23.tar.gz; do
+                            until aws s3 cp --no-progress s3://pxc-build-cache/${BUILD_TAG}/pxb24.tar.gz ./pxc/sources/pxc/results/pxb24.tar.gz; do
```

Comment:
tgz bundle has been renamed, and I think it will cause test script execution failure
https://github.com/Percona-Lab/jenkins-pipelines/blob/master/pxc/local/test-binary-pxc56#L21

Reply:
Updated the test-binary-pxc56 script.